### DISCUSSION TITLE
Send Events 📆

### DISCUSSION THREAD

**Discussion by FelixMalfait** [twentyhq/twenty#252] [Ideas]
Similar and follow-up to https://github.com/twentyhq/twenty/discussions/251
Use the Google API to send Calendar events directly from the app

**Comment by Marioviki** [comment]
Can Google Api be implement to such shop Viking Jewelry, Rings, Bracelets, Runes to Clothing?
<a href="https://vikingthunder.com/"> Viking jewelry, rings, bracelet</a>

**Comment by BOHEUS** [comment]
Related to #7008
